.github/workflows/ci.yml

```diff
@@ -54,42 +101,73 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         name: Checkout code
+
+      - name: Pull snuba CI images
+        run: |
+          docker pull ghcr.io/getsentry/snuba-ci:${{ github.sha }} || \
+            docker pull ghcr.io/getsentry/snuba-ci:${{ needs.snuba-image.outputs.branch }} || \
+            docker pull ghcr.io/getsentry/snuba-ci:latest || true
+
+      - name: Build snuba docker image for CI
+        run: |
+          docker build . \
+            --build-arg PYTHON_VERSION=3.8 \
+            -t snuba-test \
+            --cache-from ghcr.io/getsentry/snuba-ci:${{ github.sha }} \
+            --cache-from ghcr.io/getsentry/snuba-ci:${{ needs.snuba-image.outputs.branch }} \
+            --cache-from ghcr.io/getsentry/snuba-ci:latest \
+            --target testing
```

Comment:
I find it quite sad that we build almost the same Snuba image twice and then store it thrice. Can we not consolidate all this?

Reply:
Yeah, are you referring to the image here and the one built by GCB?

Reply:
Yup 🙂

Reply:
Yeah we'll do another pass at this when we/ops has GCB finalized.